### PR TITLE
Disable set log level dropdown in view only mode

### LINKED ISSUE #5714: Kiali view-only mode allows changing proxy log-level
Originally reported in https://issues.redhat.com/browse/OSSM-2371

Despite Kiali being configured as "view-only", a user can change the proxy logging level via the Workload details' Logs tab's kebab menu.  This should be prevented as it updates the existing workload configuration.

### PR DESCRIPTION
**Pull request by josunect**
Fixes #5714 

In view only mode disable the option to set the log level: 

image

Note: There is no change in kiosk mode.

### CODE REVIEW

File: handlers/proxy_logging.go [hunk 1]
```diff
@@ -8,11 +8,17 @@ import (
...
 
 	"github.com/kiali/kiali/business"
+	"github.com/kiali/kiali/config"
 )
 
 func LoggingUpdate(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
 
+	if config.Get().Deployment.ViewOnlyMode {
+		RespondWithError(w, http.StatusForbidden, "Logs cannot be edited in View only mode")
```
Comment (jshaughn) [inline handlers/proxy_logging.go:18]:
```suggestion
		RespondWithError(w, http.StatusForbidden, "Log level cannot be changed in view-only mode")
```

**Review by jshaughn — COMMENTED** [review]
Looks good, two suggested changes to text.  One is an automated suggested update, for the other, perhaps change https://github.com/kiali/kiali/pull/5717/files#diff-e6146725a1698b6ad26c3376b62100fd128467dc86a07ce2b4d07e04174d5074R588

From: `This option is disabled for pods with no proxy container.`
To: `This option is disabled for pods with no proxy container, or in view-only mode.`

**Comment by josunect** [comment]
> Looks good, two suggested changes to text. One is an automated suggested update, for the other, perhaps change https://github.com/kiali/kiali/pull/5717/files#diff-e6146725a1698b6ad26c3376b62100fd128467dc86a07ce2b4d07e04174d5074R588
> 
> From: `This option is disabled for pods with no proxy container.` To: `This option is disabled for pods with no proxy container, or in view-only mode.`

Thanks, @jshaughn ! I've committed the suggestion.

**Review by jshaughn — COMMENTED** [review]
I'm not seeing the second suggested change in the current file changes, is that intentional?

Suggested changing https://github.com/kiali/kiali/blob/344bd488f26e64d687ad018d8f2aa244a80cc160/frontend/src/pages/WorkloadDetails/WorkloadPodLogs.tsx#L588

From: `This option is disabled for pods with no proxy container.`
To: `This option is disabled for pods with no proxy container, or in view-only mode.`

**Comment by josunect** [comment]
> , or in view-only mode.

Sorry, @jshaughn ! It was not intentional. I have committed the other change. I will request the review after the checks passed.

**Review by jshaughn — APPROVED** [review]
LG2M!

**Pull request merged**